dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java

```diff
@@ -380,20 +398,24 @@ private void validateIncomingParams(final Set<String> incomingParamNames, final
      * @throws IOException
      * @throws DotDataException
      */
-    void createServiceIntegration(final FormDataMultiPart multipart, final User user)
+    List<ServiceIntegrationView> createServiceIntegration(final FormDataMultiPart multipart, final User user)
             throws IOException, DotDataException {
         final List<File> files = new MultiPartUtils().getBinariesFromMultipart(multipart);
         if(!UtilMethods.isSet(files)){
             throw new DotDataException("Unable to extract any files from multi-part request.");
         }
+        List<ServiceIntegrationView> serviceIntegrationViews = new ArrayList<>(files.size());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'serviceIntegrationViews' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4979325)